src/events/actions.py

```diff
@@ -18,11 +19,11 @@
 logger = logging.getLogger(__name__)
 
 
-class PasswordActionEvents(Object):
+class ActionEvents(Object):
     """Event handlers for password-related Juju Actions."""
```

Comment:
Shouldn't the description also change?

Reply:
Good point also - [`defc362` (#269)](https://github.com/canonical/kafka-operator/pull/269/commits/defc3627cabe0ce721ece556c141a040eb2fbec2)